subjects/DPxAI/play-with-variables/README.md

```diff
@@ -0,0 +1,93 @@
+## Play with variables
+
+> Mindful AI mode
+
+### Context
+
+Remember that if things get a little hectic at times, take the time to get closer to your peers so that you can think, share and move forward together.
+
+> Keep Going!
+
+### AI-Powered Learning Techniques
+
+**Clarification Technique:**
+
+This type of prompt encourages the AI to explain a concept in detail, helping you gain a deeper understanding.
+
+> Find the examples across the subject ;)
+
+## Concepts
```

Comment:
This will never show as the platform markdown ignores the headers with two #:
![image](https://github.com/user-attachments/assets/1565a916-18cd-48c6-ae78-7b340f496be6)